### ISSUE TITLE
Plugin [id: 'org.openjfx.javafxplugin', version: '0.0.8'] was not found

### ISSUE THREAD

**Issue by schildbach** [bitcoinj/bitcoinj#2300]
We do have this issue with all our GitHub Actions builds:

```
org.gradle.api.plugins.UnknownPluginException: Plugin [id: 'org.openjfx.javafxplugin', version: '0.0.8'] was not found in any of the following sources:

- Gradle Core Plugins (plugin is not in 'org.gradle' namespace)
- Plugin Repositories (could not resolve plugin artifact 'org.openjfx.javafxplugin:org.openjfx.javafxplugin.gradle.plugin:0.0.8')
  Searched in the following repositories:
    Gradle Central Plugin Repository
```

At the moment, I assume that we don't have this in our hands, but I thought I'll file it anyway.

**Comment by schildbach** [comment]
GitLab CI works fine.

**Comment by msgilligan** [comment]
Maybe we should try bumping the version.

**Comment by msgilligan** [comment]
@schildbach I haven't seen this failure. Was this intermittent?

I created PR #2306 which upgrades to the latest. This might help, but we should update anyway.

**Comment by schildbach** [comment]
The issue was just intermittent, but I merged #2306 anyway.
